wikihow2zim/entrypoint.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import argparse
+
+from .constants import NAME, SCRAPER, URLS, setDebug, getLogger
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        prog=NAME,
+        description="Scraper to create ZIM files wikihow articles",
+    )
+    #["en", "es", "pt", "it", "fr", "ru", "de", "zh", "nl", "cz", "id", "jp", "hi", "th", "ar", "vn"],
+    parser.add_argument(
+        "--language",
+        choices=list(URLS.keys()),
+        default="en",
+        help="The language is specified using the language code. "
+                "The default language is English. "
+                "You may choose from one of the following languages that wikihow "
+                "is available in: "
+                "English-en, "
+                "Spanish-es, "
+                "Portugese-pt, "
+                "Italian-it, "
+                "French-fr, "
+                "Russian-ru, "
+                "German-de, "
+                "Chinese-zh, "
+                "Netherlands-nl, "
+                "Chech-cz, "
+                "Indonesian-id, "
+                "Japanese-jp, "
+                "Hindi-hi, "
+                "Thai-th, "
+                "Arabic-ar, "
+                "Vietnamese-vn, "
+                "Korean-ko, "
+                "Turkish-tr "
+    )
+
+    parser.add_argument(
+        "--name",
+        help="ZIM name. Used as identifier and filename (date will be appended)",
+        required=True,
```

Comment:
I think we can drop the requirement as the language will allow us to compute a standard Name like`wikihow_{lang}_all`